test/app-luatest/shutdown_test.lua

```diff
@@ -275,3 +275,37 @@ g_vinyl.test_shutdown_vinyl_dump_during_snapshot = function(cg)
     fiber.sleep(3)
     test_no_hang_on_shutdown(cg.server)
 end
+
+local g_engine = t.group('engine', {{engine = 'memtx'}, {engine = 'vinyl'}})
```

Comment:
Should be in `engine-luatest`. And vinyl tests should be in `vinyl-luatest`.

Reply:
Done.